api/tools/src/main/java/org/pmiops/workbench/tools/ManageClusters.java

```diff
@@ -91,6 +101,42 @@ private static void listClusters(String apiUrl) throws IOException, ApiException
     System.out.println(String.format("listed %d clusters", count.get()));
   }
 
+  private static void describeCluster(
+      String apiUrl, String workbenchProjectId, String workbenchServiceAccount, String clusterId)
+      throws IOException, ApiException {
+    String[] parts = clusterId.split("/");
+    if (parts.length != 2) {
+      System.err.println(
+          String.format(
+              "given cluster ID '%s' is invalid, wanted format 'project/clusterName'", clusterId));
+      return;
+    }
+
+    // Leo's getCluster API swagger tends to be outdated; issue a raw getCluster request to ensure
+    // we get all available information for debugging.
+    ClusterApi client = newApiClient(apiUrl);
+    com.squareup.okhttp.Call call = client.getClusterCall(parts[0], parts[1], null, null);
+    ApiResponse<Object> resp = client.getApiClient().execute(call, Object.class);
+
+    // Parse the response as well so we can log specific structured fields.
+    Cluster cluster = PRETTY_GSON.fromJson(PRETTY_GSON.toJson(resp.getData()), Cluster.class);
+
+    System.out.println(PRETTY_GSON.toJson(resp.getData()));
+    System.out.printf("\n\nTo inspect logs in cloud storage, run the following:\n\n");
```

Comment:
should this specify that you should run this on your cluster?

Reply:
No - you aren't supposed to do that (and permissions wise, this should probably be impossible to run on a workbench Leo cluster). This is intended for execution on a dev workstation.

Reply:
Oh, then I missed the point of why we're printing out stuff to be copied and executed later. The idea is that the logs are protected at a higher level than you necessarily have when you're running this, and/or they're too voluminous to dump to the terminal.

Reply:
I've added a bit more instruction to this output. If you decide to run them, these commands get you to effectively a debug session. Human decisions are required because there are multiple log files you might want to look at, and we don't want to depend on the Leo-specific directory structure / naming.

After typing this out, it would good to automate more of this. Once I have https://precisionmedicineinitiative.atlassian.net/browse/PD-4740, I'll revisit this and glob out all the files in the directory, then give a simple commandline someone could use to copy/cat the logs if desired.